tox.ini

```diff
@@ -8,6 +8,10 @@ commands =
 usedevelop = True
 extras =
 	test
+	cover
+	type
+	check
+	enabler
```

Comment:
Probably best is to keep it in the same order as in pyproject.toml

Reply:
Ugh, now I'm conflicted. I was thinking if we're going to be explicit about order, we should list the optional dependencies in alphabetical order, but that means pushing 'test' near the bottom and putting 'doc' somewhere in the middle, but those two are kind-of special (primal). If I choose to re-order everything, it's going to lead to some annoying merge conflicts.

In bc5db71, I've settled on the worst compromise - putting 'test' and 'doc' first, then doing the rest alphabetically, though I'm not sure I love it.

Where's a TOML linter to remove ambiguity when you need one?

Reply:
I found https://github.com/TimonPost/cargo-toml-format, but it's made primarily for cargo files. I'd probably try it if it were published, but it's only available when built from source.